sources/common/infoschema.go

```diff
@@ -53,31 +53,33 @@ type SchemaAndName struct {
 
 // FkConstraint contains foreign key constraints
 type FkConstraint struct {
-	Name    string
-	Table   string
-	Refcols []string
-	Cols    []string
+	Name     string
+	Table    string
+	Refcols  []string
+	Cols     []string
+	OnDelete string
+	OnUpdate string
 }
 
 type InfoSchemaInterface interface {
 	GenerateSrcSchema(conv *internal.Conv, infoSchema InfoSchema, numWorkers int) (int, error)
 	ProcessData(conv *internal.Conv, infoSchema InfoSchema, additionalAttributes internal.AdditionalDataAttributes)
 	SetRowStats(conv *internal.Conv, infoSchema InfoSchema)
 	processTable(conv *internal.Conv, table SchemaAndName, infoSchema InfoSchema) (schema.Table, error)
-	GetIncludedSrcTablesFromConv(conv *internal.Conv) (schemaToTablesMap map[string]internal.SchemaDetails, err error) 
+	GetIncludedSrcTablesFromConv(conv *internal.Conv) (schemaToTablesMap map[string]internal.SchemaDetails, err error)
 }
-type InfoSchemaImpl struct {}
```

Comment:
It looks like the lint is off. Please install the golint plugin in vscode (or run int manually) so that everytime you save locally, it standardizes the formatting.